phylogenetic/rules/prepare_sequences.smk

```diff
@@ -79,7 +87,7 @@ rule align:
     """
     input:
         sequences = "results/{build}/filtered.fasta",
-        reference = lambda w: config["build_params"][w.build]["files"]["reference"],
+        reference = config_path("align", "reference"),
```

Comment:
Why not use `resolve_config_path()` directly instead of creating a new function `config_path()` which splits the definition of config keys between a function and its parameters?

```suggestion
        reference = lambda w: resolve_config_path(config["build_params"][w.build]["align"]["reference"], PHYLO_DIR),
```

It's more verbose, but I think that's a good thing because it provides a more clear mapping to values in `config.yaml`.

Reply:
I'm going to keep them for now. Hopefully they won't be there for long as we can get rid of all this code using the approach of https://github.com/nextstrain/ebola/commit/a74f20e or by implementing YAML-config-in-augur-commands.